index.html

```diff
@@ -2,9 +2,197 @@
+                <h3>Web Development</h3>
+                <p>Modern websites need to be responsive and fully functional at all sizes. We develop all of our sites with a mobile first approach and ensure that our code quality is clean, organized and performant. Already have a design but need it developed? We’re here to help!</p>
+
+            </aside>
+
+
+            <aside class="right">
+                <img class="small-icon" src="/assets/icon_ux.svg" alt="user">
+                <h3>User Experience</h3>
+                <p>In order to build successful products you have to understand the user. We create user personas, wireframes and prototypes to test with users. This allows us to gather feedback which helps to shape the product and can quickly eliminate potential risks – saving you time and money.</p>
+            </aside>
+
+            <aside class="left">
+                <img class="small-icon" src="/assets/icon_graphic-design.svg" alt="graph-design">
+                <h3>Logo Design + Branding</h3>
+                <p>Nike, Apple, Coca-Cola, Starbucks. What do all of these brands have in common? Easily recognizable logos and brand identity. The best brands stand out from the crowd by having a clear and unique point of view. We will work with you to understand your vision and help create a unique brand identity and style guide for your company.</p>
+            </aside>
+
+            <aside class="middle">
+                <img class="small-icon" src="/assets/icon_seo.svg" alt="seo-image">
+                <h3>SEO</h3>
+                <p>Do you want your site to rank higher in search results? Google is constantly changing their algorithms for how they rank pages. Don’t worry about constantly playing catch-up. We will make sure your website it optimized, providing you the best opportunities for organic search exposure.</p>
+            </aside>
+
+            <aside class="right">
+                <img class="small-icon" src="/assets/icon_copywriting.svg" alt="user">
+                <h3>Copy Writing</h3>
+                <p>Brand messaging needs to be clear, direct and easy to understand to be effective. A unique tone of voice will help you stand out from the crowd and build brand loyalty. We work with you to establish your brand bible and ensure your messaging is focused and targeted.</p>
+            </aside>
+
+            </div>
+
+        </article>
+
+    </section>
+</main>
+
+
+    <footer class="footer-outer-container">
+
+        <article class="footer-inner-container">
+
+            <div class="footer-paragraph">
+                <h1>Let's work together</h1>
+                <p>Have a question? Need a quote? Let us know!</p>
+
+            </div>
+
+            <div class="footer-form-container">
+                    <form>
+                        <label for="name">Name</label><br>
+                        <input id="name" name="name" type="text" placeholder="Name"><br>
+                <br>
```

Comment:
Self-closing tags gebruiken
```suggestion
                <br />
```